cpp/autosar/src/rules/M5-2-10/IncrementAndDecrementOperatorsMixedWithOtherOperatorsInExpression.ql

```diff
@@ -17,7 +17,13 @@
 import cpp
 import codingstandards.cpp.autosar
 
-from CrementOperation cop, Operation op, string name
+class ArithmeticOperation extends Operation {
```

Comment:
Optionally put this in a common module for reuse.

Reply:
Done.